core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/imagecatalog/base/ImageCatalogV4Base.java

```diff
@@ -29,13 +29,28 @@ public class ImageCatalogV4Base {
     @ValidImageCatalog
     private String url;
 
+    /**
+     * @deprecated data owner of any user is UMS, creator should not be stored and used anywhere, since user of creator can leave the given company
+     * and can become invalid, usage of it can be error prune
+     */
+    @Deprecated
     @ApiModelProperty(ModelDescriptions.CREATOR)
```

Comment:
Could you please include it in the @ApiModelPropery descriptions (all occurrences) as well?

Reply:
sure